src/App.tsx

```diff
@@ -27,6 +28,7 @@ function App() {
             <Route path="/" element={<HomePage />} />
             <Route index element={<HomePage />} />
             <Route path="bounty" element={<BountyPage />} />
+            <Route path="onboarding" element={<OnboardingPage />} />
             <Route path="/creator/:creatorId" element={<PurchasePage />} />
             <Route path="*" element={<NotFoundPage />} />
           </Routes>
```

Comment:
can I make a suggestion, please define the routes as a constant file somewhere its a bit easier to maintain.

e.g.
export const MenuItems = [
    {
        title: "Home",
        url: '/',
        cName: 'nav-links',
    }

Reply:
will add in next PR -- i'm adding a few pages in another PR and will clean up routes